app/graphql/types/provider_enum.rb

```diff
@@ -1,6 +1,6 @@
 module Types
   class ProviderEnum < Types::BaseEnum
-    ::ComputeResource.supported_providers.keys.each do |provider|
```

Comment:
Now I wonder what the point of `supported_providers` is. Feels like this is more like `builtin_providers`.